src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -325,6 +410,19 @@ public void updateCommissioningNetworkCredentials(NetworkCredentials networkCred
     updateCommissioningNetworkCredentials(deviceControllerPtr, networkCredentials);
   }
 
+  /**
+   * Update the ICD registration information held by the commissioner for the current commissioning
+   * session.
+   *
+   * <p>Its expected that this method will be called in response the onICDRegistrationInfoRequired
```

Comment:
<p> typo?

Reply:
Is this a comment related to "<p>"?
It was written in the same format as updateCommissioningNetworkCredentials above.
https://github.com/project-chip/connectedhomeip/blob/711ca50d4f2937b62799e89dd80218a4f94ff3be/src/controller/java/src/chip/devicecontroller/ChipDeviceController.java#L319

Could it be a different reason than I thought?

Reply:
not sure "<p>" meaning...remove it?